common/install_rocm_miopen_rpm.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -ex
+
+# Create the tmp dir to extract into
+EXTRACTDIR_ROOT=/extract_miopen_rpm
+mkdir -p ${EXTRACTDIR_ROOT}
+echo "Creating temporary directory for rpm download..."
+
+# Fail if rpm source is not available
+if ! wget -P ${EXTRACTDIR_ROOT} ${MIOPEN_RPM_SOURCE}; then
+  echo 'ERROR: Failed to download MIOpen package.'
+  exit 1
+fi
+echo "MIOpen package download complete..."
+
+# Extract rpm in EXTRACT_DIR
+cd ${EXTRACTDIR_ROOT}
+miopen_rpm=$(ls *.rpm)
+rpm2cpio ${miopen_rpm} | cpio -idmv 
+
+# Copy libMIOpen.so.1 over existing
+source_file=$(ls opt/rocm-*/lib/libMIOpen.so.1.0*)
```

Comment:
May not be an immediate issue, but the hardcoding of the MIOpen so version bugs me :)